packages/wordpress-external-dependencies-plugin/index.js

```diff
@@ -0,0 +1,23 @@
+const { ExternalsPlugin } = require( 'webpack' );
+
+class WordPressExternalDependenciesPlugin extends ExternalsPlugin {
+	constructor() {
+		super( 'global', [ wpExternals ] );
+	}
+}
+
+function wordpressRequire( request ) {
+	// @wordpress/components -> [ @wordpress, components ]
```

Comment:
I like `@wordpress/api-fetch` example more here because it also shows also how we camelCase in this function.

Example doc block: https://github.com/WordPress/gutenberg/blob/6e340d6a4dbd0daa9d459e09592200ddcb676eb2/packages/scripts/config/webpack.config.js#L13-L28